pandas/core/internals/managers.py

```diff
@@ -1396,7 +1394,7 @@ def _slice_take_blocks_ax0(self, slice_or_indexer, fill_tuple=None):
                             blklocs[mgr_locs.indexer],
                             axis=0,
                             new_mgr_locs=mgr_locs,
-                            fill_tuple=None,
+                            fill_value=lib.no_default,
```

Comment:
should this be removed? isn't it an anti-pattern to explicitly pass this.

Reply:
good idea